src/telemetry/telemetry.c

```diff
@@ -937,6 +1025,17 @@ build_telemetry_report()
 	add_replication_telemetry(parse_state);
 	pushJsonbValue(&parse_state, WJB_END_OBJECT, NULL);
 
+	key.type = jbvString;
+	key.val.string.val = REQ_ACCESS_METHODS;
+	key.val.string.len = strlen(REQ_ACCESS_METHODS);
+	(void) pushJsonbValue(&parse_state, WJB_KEY, &key);
+	add_query_result_dict(parse_state,
```

Comment:
Nit: Other functions in this file (e.g., `add_job_stats_by_job_type`) use SQL in uppercase.

Reply:
Switched to uppercase.